liquibase-standard/src/main/java/liquibase/util/ShowSummaryUtil.java

```diff
@@ -361,4 +381,14 @@ private static void writeToLog(String message) {
                 .filter(s -> !StringUtil.isWhitespace(s))
                 .forEach(Scope.getCurrentScope().getLog(ShowSummaryUtil.class)::info);
     }
+
+    /**
+     * Container to handle sharing update summary message between different services
+     */
+    @Data
+    @Beta
+    public static class UpdateSummaryDetails {
```

Comment:
Let me know if you think this approach is legitimate, I'll throw this class into it's own file and clean things up a bit more if so.

Reply:
Seems good to me